collections/_general/tag.html

```diff
@@ -0,0 +1,254 @@
+    </div>
+  </div>
+
+  <div class="control">
+    <div class="sgds-tags has-addons">
+      <span class="sgds-tag is-dark">build</span>
+      <span class="sgds-tag is-success">passing</span>
+    </div>
+  </div>
+
+  <div class="control">
+    <div class="sgds-tags has-addons">
+      <span class="sgds-tag is-dark">chat</span>
+      <span class="sgds-tag is-primary">telegram</span>
+    </div>
+  </div>
+</div>
+{% endcapture %}
+
+<h2>Overview</h2>
+<p>
+    Tags can be used to label or classify content.
+</p>
+
+<hr />
+
+<!-- TODO When to use and consideration-->
+
+<h3>Tag colours</h3>
+<div class="sgds-example-others">
+    {{tag}}
+</div>
+{%- highlight html -%}{{tag}}{%- endhighlight -%}
+<button class="sgds-button clipboard-btn is-primary is-outlined" data-clipboard-target='.highlight0'>Copy
+  snippet</button>
+
+<h4>Light variants</h4>
+
+<p>Add the <code>.is-light</code> class for tag with less emphasis.</p>
+
+<div class="sgds-example-others">
+    {{tag_is_light}}
+</div>
+{%- highlight html -%}{{tag_is_light}}{%- endhighlight -%}
+<button class="sgds-button clipboard-btn is-primary is-outlined" data-clipboard-target='.highlight1'>Copy
+  snippet</button>
+
+<hr />
+
+<h3>Groups of tags</h3>
+<p>Wrap tags with an element with the <code>.sgds-tags</code> class to easily display a list of tags that are
+  vertically centered and automatically wrap lines.</p>
+<div class="sgds-example-others">
+    <div class="row">
+        <div class="col is-12-mobile is-8-tablet">
+            {{tags_list}}
+        </div>
+    </div>
+</div>
+{%- highlight html -%}{{tags_list}}{%- endhighlight -%}
+<button class="sgds-button clipboard-btn is-primary is-outlined" data-clipboard-target='.highlight2'>Copy
+  snippet</button>
+  
+<hr />
+
+<h3>Tag sizes</h3>
+<div class="sgds-example-others">
+    {{tag_sizes}}
+</div>
+{%- highlight html -%}{{tag_sizes}}{%- endhighlight -%}
+<button class="sgds-button clipboard-btn is-primary is-outlined" data-clipboard-target='.highlight3'>Copy
+  snippet</button>
+
+<h4>Size entire groups of tags</h4>
+
+<p>Change tags sizes for entire groups of tags by wrapping
+  tags with an element with <code>.sgds-tags .are-medium</code> classes.</p>
+<div class="sgds-example-others">
+    {{tags_are_medium}}
+</div>
+{%- highlight html -%}{{tags_are_medium}}{%- endhighlight -%}
+<button class="sgds-button clipboard-btn is-primary is-outlined" data-clipboard-target='.highlight4'>Copy
+  snippet</button>
+
+<hr>
+
+<h3>Tags with addons</h3>
+<p>Join tags with the <code>has-addons</code> class.</p>
+<div class="sgds-example-others">
+    {{tags_add_on}}
+</div>
+{%- highlight html -%}{{tags_add_on}}{%- endhighlight -%}
+<button class="sgds-button clipboard-btn is-primary is-outlined" data-clipboard-target='.highlight5'>Copy
+  snippet</button>
+
+<h4>Group tags with addons</h4>
+<p>Group the list of addons tag with <code>field is-grouped is-grouped-multiline</code></p>
```

Comment:
Can add some description for what `field is-grouped is-grouped-multiline` does?
Is the `control` class needed?
How does this differ from what the `sgds-tags` wrapper class does?

Reply:
1. `field is-grouped is-grouped-multiline` is use to group multiple `sgds-tags has-addons` and uses flex-wrap to prevent them from overflowing out of the container's width whereas `sgds-tags` is for grouping default tag only.

2. `control` is use to add spacing inbetween sgds-tags with addons.